icgcget/clients/download_client.py

```diff
@@ -145,16 +154,15 @@ def parse_test_ex(ex, forbidden, not_found):
         else:
             return 0
 
-    def prepend_docker_args(self, args, mnt=None, envvars=None):
+    def prepend_docker_args(self, args, mnt=None, envvars=None, cidname=None):
```

Comment:
You are later checking if `envvars` is `None` and assigning it as an empty dict. Why not just do that here as a default value?